### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -420,6 +420,10 @@ GEM
     kaminari (0.17.0)
       actionpack (>= 3.0.0)
       activesupport (>= 3.0.0)
+    kaminari-actionview (1.0.1)
```
Comment (jcoyne) [inline Gemfile.lock]: These 4 lines shouldn't be here. Please remove. see https://github.com/bundler/bundler/issues/5364

**Comment by cbeer** [comment]
Why are we pinned to `kaminari_route_prefix ~> 0.0.1`? v0.1 was released with support for kaminari 1.0.

**Comment by jcoyne** [comment]
@cbeer  a fine question. I opened a PR: https://github.com/projecthydra-labs/hyrax/pull/539

File: Gemfile.lock [hunk 2]
```diff
@@ -420,6 +420,10 @@ GEM
     kaminari (0.17.0)
```
Comment (jeremyf) [inline Gemfile.lock]: Should kaminari still be pinned to 0.17.0? I noticed that there were breaking changes in the pager methods (`#num_pages` becamse `#total_pages`). The breaks will only be noticed if there is direct interaction with the pager via Hydra code.

https://github.com/ndlib/sipity/commit/ab419af73a1662030fa01ce621cf5e20687e0c8b

**Comment by mjgiarlo** [comment]
@jcoyne @jeremyf Updated.

**Comment by jeremyf** [comment]
@mjgiarlo 

```console
1) Admin Dashboard as an administrator displays the add-users-to-groups page without the hidden form field
     Failure/Error: <%= paginate @users %>
     
     ActionView::Template::Error:
       Current page was incalculable. Perhaps you called .per(0)?
     # ./app/views/admin/groups/users.html.erb:85:in `_app_views_admin_groups_users_html_erb___1364763279164377017_126139660'
     # /home/travis/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
     # ./app/controllers/admin/group_users_controller.rb:7:in `index'
     # /home/travis/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/webrick/httpserver.rb:140:in `service'
     # /home/travis/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/webrick/httpserver.rb:96:in `run'
     # /home/travis/.rvm/rubies/ruby-2.4.0/lib/ruby/2.4.0/webrick/server.rb:290:in `block in start_thread'
     # ------------------
     # --- Caused by: ---
     # ZeroDivisionError:
     #   divided by 0
     #   ./app/views/admin/groups/users.html.erb:85:in `_app_views_admin_groups_users_html_erb___1364763279164377017_126139660'
```

**Comment by mjgiarlo** [comment]
@jcoyne @jeremyf @cbeer Do any of y'all know offhand what kaminari we should be including here? I'm not sure what I need to do at this point to get Hyku's Hyrax dep updated.

**Comment by mjgiarlo** [comment]
:+1: 

Take another look, @jeremyf?

**Pull request merged**